spec/models/podcast_episode_spec.rb

```diff
@@ -5,6 +5,9 @@
 
   it "accepts valid podcast episode" do
     expect(podcast_episode).to be_valid
+    perform_enqueued_jobs do
```

Comment:
These lines seem to not be related to the existing test. Busting itself is tested in the job specs, so it makes sense to only test that the job was enqueued in a separate spec in my opinion.

Reply:
I updated the tests here and in the job spec.